Packs/RiskIQDigitalFootprint/Integrations/RiskIQDigitalFootprint/RiskIQDigitalFootprint.py

```diff
@@ -409,27 +409,33 @@ def get_asset_connections_outputs(resp: Dict[str, Any],
         'IP_ADDRESS': get_standard_context_ip,
         'PAGE': get_standard_context_url
     }
-    asset_connections_custom_context: List[Dict[str, Any]] = []
-    asset_connections_standard_context = []
+    command_results: List[CommandResults] = []
 
     hr += '### CONNECTED ASSETS\n'
     for asset_type in total_elements:
         if total_elements.get(asset_type) and total_elements.get(asset_type) != 0:
-            asset_connections_hr: List[Dict[str, Any]] = []
             for asset_details in resp.get(asset_type, {}).get('content', []):
-                asset_connections_custom_context.append(asset_details)
+                asset_connections_standard_context = None
+                asset_connections_custom_context = asset_details
                 if asset_type in asset_type_standard_context_map:
-                    asset_connections_standard_context.append(asset_type_standard_context_map[asset_type]
-                                                              (asset_details))
-                asset_connections_hr.append(get_asset_connections_hr(asset_details))
+                    asset_connections_standard_context = asset_type_standard_context_map[asset_type](asset_details)
+                asset_connections_hr = tableToMarkdown(f'{asset_type.replace("_", " ").capitalize()} Details',
+                                                       get_asset_connections_hr(asset_details),
+                                                       ['Name', 'State', 'First Seen (GMT)', 'Last Seen (GMT)',
+                                                        'Tag(s)'], removeNull=True)
+                custom_ec = createContext(remove_empty_entities(asset_connections_custom_context), removeNull=True)
+                command_results.append(CommandResults(
+                    outputs_prefix='RiskIQDigitalFootprint.Asset',
+                    outputs_key_field='uuid',
+                    outputs=custom_ec,
+                    indicator=asset_connections_standard_context,
+                    readable_output=asset_connections_hr,
+                    raw_response=resp
```

Comment:
I think resp contains all the data and will load the DB
```suggestion
                    raw_response=asset_details
```

Reply:
The raw_response isn't being loaded to the DB, but since resp is returned as a raw_response elsewhere, it's ok to change this.